[2021-08-12 13:22:53,255] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: pipeline_try.thursday_Task 2021-08-12T13:22:46.061693+00:00 [queued]>
[2021-08-12 13:22:53,279] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: pipeline_try.thursday_Task 2021-08-12T13:22:46.061693+00:00 [queued]>
[2021-08-12 13:22:53,280] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 13:22:53,281] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2021-08-12 13:22:53,281] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-12 13:22:53,294] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): thursday_Task> on 2021-08-12T13:22:46.061693+00:00
[2021-08-12 13:22:53,303] {standard_task_runner.py:52} INFO - Started process 2812 to run task
[2021-08-12 13:22:53,307] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'pipeline_try', 'thursday_Task', '2021-08-12T13:22:46.061693+00:00', '--job-id', '102', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/pipeline_try1.py', '--cfg-path', '/tmp/tmpdq7cwayd', '--error-file', '/tmp/tmpn0oev6zg']
[2021-08-12 13:22:53,312] {standard_task_runner.py:77} INFO - Job 102: Subtask thursday_Task
[2021-08-12 13:22:53,380] {logging_mixin.py:104} INFO - Running <TaskInstance: pipeline_try.thursday_Task 2021-08-12T13:22:46.061693+00:00 [running]> on host 99300daf4d26
[2021-08-12 13:22:53,454] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=ameysonje242000@gmail.com
AIRFLOW_CTX_DAG_OWNER=amey
AIRFLOW_CTX_DAG_ID=pipeline_try
AIRFLOW_CTX_TASK_ID=thursday_Task
AIRFLOW_CTX_EXECUTION_DATE=2021-08-12T13:22:46.061693+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-08-12T13:22:46.061693+00:00
[2021-08-12 13:22:53,456] {logging_mixin.py:104} INFO - From thursday_Task, so its thursday today
[2021-08-12 13:22:53,456] {python.py:151} INFO - Done. Returned value was: ('httpposttry', 'httpgettry')
[2021-08-12 13:22:53,491] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=pipeline_try, task_id=thursday_Task, execution_date=20210812T132246, start_date=20210812T132253, end_date=20210812T132253
[2021-08-12 13:22:53,528] {taskinstance.py:1265} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2021-08-12 13:22:53,561] {local_task_job.py:149} INFO - Task exited with return code 0
